lib/autoproj/cli/base.rb

```diff
@@ -176,11 +176,13 @@ def self.validate_options(args, options)
                     debug: false,
                     color: TTY::Color.color?,
                     progress: TTY::Color.color?,
-                    parallel: nil
+                    parallel: nil,
+                    interactive: true
```

Comment:
Shouldn't this be `nil`?

Reply:
Right and I have to change the default in the cli/main.rb as well.